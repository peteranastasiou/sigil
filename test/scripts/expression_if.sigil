
var arg1 = "hello";

var a = if arg1 == "hello" { print("A1"); "A2" } else { "A3" };
print(a);   # should be A2

{
    var d = if 7>6 { print("B1"); if false {"B2"} else {"B3"} } else { print("B4"); "B5" };
    print(d);   # should be 2
}
