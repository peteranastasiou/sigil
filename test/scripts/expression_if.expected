A1
A2
B1
B3
